README.md

```diff
@@ -24,25 +24,99 @@ Change directory to the package under development. Note: an integration is a spe
 
 ```bash
 cd integrations
-cd package/my-integration
+cd package/my-package
 ```
 
-Run the `help` command and see available actions:
+Run the `help` command and see available commands:
 
 ```bash
 elastic-package help
 ```
 
-## GitHub authorization
+## Commands
+
+`elastic-package` currently offers the commands listed below. 
+
+Some commands have a _global context_, meaning that they can be executed from anywhere and they will have the 
+same result. Other commands have a _package context_; these must be executed from somewhere under a package's
+root folder and they will operate on the contents of that package.
+
+For more details on a specific command, run `elastic-package help <command>`.
+
+### `elastic-package help`
+
+Use this command to get a listing of all commands available under `elastic-package` and a brief
+description of what each command does.
+
+_Context: global_
+
+
+### `elastic-package build`
+
+Use this command to build a package. TODO: what is the purpose of building a package — when would
+a package developer want/need to run this command?
+
+_Context: package_
+
+
+### `elastic-package check`
+
+Use this command to run the `format`, `lint`, and `build` commands all at once, in that order.
+
+_Context: package_
 
-The `promote` subcommand requires access to the GitHub API to open pull requests or check authorized account data.
+
+### `elastic-package format`
+
+Use this command to format the contents of a package.
+
+_Context: package_
+
+
+### `elastic-package lint`
+
+Use this command to validate the contents of a package using the 
+[package specification](https://github.com/elastic/package-spec).
+
+_Context: package_
+
+
+### `elastic-package promote`
+
+Use this command to promote a package to the Snapshot Package Registry.
+
+_Context: package_
```

Comment:
@mtojek I think this should be `global`, right, since the tool lets the user select multiple packages at once?

Reply:
Correct, I missed this one. It should be `global`.

Reply:
Fixed in 291401e.